cmake/packaging/linux.cmake

```diff
@@ -28,6 +28,10 @@ endif()
 set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/postinst")
 set(CPACK_RPM_POST_INSTALL_SCRIPT_FILE "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/postinst")
 
+# Apply setcap for RPM
+# https://github.com/coreos/rpm-ostree/discussions/5036#discussioncomment-10291071
+set(CPACK_RPM_USER_FILELIST "%caps(cap_sys_admin+p) %{_bindir}/${CPACK_PACKAGE_FILE_NAME}")
```

Comment:
- file name is lowercase
- probably need to append the list instead of set it directly